Assets/MixedRealityToolkit/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -36,7 +45,7 @@ public class MixedRealityProjectConfiguratorWindow : EditorWindow
         private readonly GUIContent LaterButtonContent = new GUIContent("Later", "Do not show this popup notification until next session");
         private readonly GUIContent IgnoreButtonContent = new GUIContent("Ignore", "Modify this preference under Edit > Project Settings > MRTK");
 
-        private bool showConfigurations = false;
+        private bool showConfigurations = true;
```

Comment:
Why show true by default? This is kind of like the "Use recommended settings" and then you click expand to actually see the entire change list

Reply:
i found that i was always expanding.... i can put it back :)